### PR TITLE
add Dutch translations

### PR DESCRIPTION
**Pull request by bradford-hamilton**

### CODE REVIEW

File: app/_locales/nl/messages.json [hunk 1]
```diff
@@ -0,0 +1,819 @@
...
+    "message": "We berekenen de voorgestelde gasprijzen op basis van succespercentages van het netwerk."
+  },
+  "gasPriceRequired": {
+    "message": "Gasprijs vereist"
+  },
+  "getEther": {
+    "message": "Krijg Ether"
+  },
+  "getEtherFromFaucet": {
+    "message": "Haal Ether uit een kraan voor de $ 1",
```
Comment (kumavis) [inline app/_locales/nl/messages.json]: this is a variable, needs to be `$1`

File: app/_locales/nl/messages.json [hunk 2]
```diff
@@ -0,0 +1,819 @@
...
+    "description": "Geeft de netwerknaam voor Ether-kraan weer"
+  },
+  "greaterThanMin": {
+    "message": "moet groter zijn dan of gelijk zijn aan $ 1.",
```
Comment (kumavis) [inline app/_locales/nl/messages.json]: this is a variable, needs to be `$1`

File: app/_locales/nl/messages.json [hunk 3]
```diff
@@ -0,0 +1,819 @@
...
+    "description": "formaat voor het importeren van een account"
+  },
+  "kovan": {
+    "message": "Kovan-testnetwerk"
+  },
+  "knowledgeDataBase": {
+    "message": "Bezoek onze Knowledge Base"
+  },
+  "lessThanMax": {
+    "message": "moet kleiner zijn dan of gelijk zijn aan $ 1.",
```
Comment (kumavis) [inline app/_locales/nl/messages.json]: this is a variable, needs to be `$1`

File: app/_locales/nl/messages.json [hunk 4]
```diff
@@ -0,0 +1,819 @@
...
+    "message": "Je saldo"
+  },
+  "balanceIsInsufficientGas": {
+    "message": "Onvoldoende saldo voor huidig ​​gastotaal"
+  },
+  "beta": {
+    "message": "BETA"
+  },
+  "betweenMinAndMax": {
+    "message": "moet groter zijn dan of gelijk zijn aan $ 1 en kleiner dan of gelijk aan $ 2.",
```
Comment (kumavis) [inline app/_locales/nl/messages.json]: this is a variable, needs to be `$1` and `$2`

File: app/_locales/nl/messages.json [hunk 5]
```diff
@@ -0,0 +1,819 @@
...
+    "message": "Uw DEN is uw wachtwoord-gecodeerde opslag binnen MetaMask."
+  },
+  "deposit": {
+    "message": "Storting"
+  },
+  "depositBTC": {
+    "message": "Stort uw BTC op het onderstaande adres:"
+  },
+  "depositCoin": {
+    "message": "Stort uw $ 1 op het onderstaande adres",
```
Comment (kumavis) [inline app/_locales/nl/messages.json]: `$1`

**Review by kumavis — CHANGES_REQUESTED** [review]

File: app/_locales/nl/messages.json (hunk 1 above)
Comment (bradford-hamilton) [reply app/_locales/nl/messages.json]: Missed these, sorry!

File: app/_locales/nl/messages.json (hunk 2 above)
Comment (bradford-hamilton) [reply app/_locales/nl/messages.json]: Missed these, sorry!

File: app/_locales/nl/messages.json (hunk 3 above)
Comment (bradford-hamilton) [reply app/_locales/nl/messages.json]: Missed these, sorry!

File: app/_locales/nl/messages.json (hunk 4 above)
Comment (bradford-hamilton) [reply app/_locales/nl/messages.json]: Missed these, sorry!

File: app/_locales/nl/messages.json (hunk 5 above)
Comment (bradford-hamilton) [reply app/_locales/nl/messages.json]: Missed these, sorry!

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
Thank you 🎉 🎉 
@bradford-hamilton Please join our translator team https://github.com/MetaMask/

**Pull request merged**